### PR TITLE
WIP - Load Data in Model Builder How-To

### PR DESCRIPTION
**Pull request by luisquintanilla**
How-To guide to load data in Model Builder

[AB#1622265](https://dev.azure.com/mseng/677da0fb-b067-4f77-b89b-f32c12bb8617/_workitems/edit/1622265)

### CODE REVIEW

File: docs/machine-learning/how-to-guides/load-data-model-builder.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+#Customer intent: As a developer, I want to load data in Model Builder
+---
+
+# Load data in Model Builder
+
+Learn how to load your data into Model Builder to train machine learning algorithms.
+
+## Limitations
+
+Training dataset limit is 1GB.
```
Comment (Youssef1313) [inline docs/machine-learning/how-to-guides/load-data-model-builder.md]:
```suggestion
- Training dataset limit is 1GB.
```

File: docs/machine-learning/how-to-guides/load-data-model-builder.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+SQL Server has a limit of 100 thousand rows.
```
Comment (Youssef1313) [inline docs/machine-learning/how-to-guides/load-data-model-builder.md]:
```suggestion
- SQL Server has a limit of 100 thousand rows.
```

File: docs/machine-learning/how-to-guides/load-data-model-builder.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+## Load data from a file
+
+Model builder supports loading data from a file in comma-delimited and tab-demlimited formats.
```
Comment (Youssef1313) [inline docs/machine-learning/how-to-guides/load-data-model-builder.md]: To be consistent with other occurrences of Model Builder.

```suggestion
Model Builder supports loading data from a file in comma-delimited and tab-demlimited formats.
```

File: docs/machine-learning/how-to-guides/load-data-model-builder.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+## Load data from a database
+
+Model Builder supports loading data from local and remote databases. 
+
+## Load image data
+
+Model Builder expects image data to be in a specific format.
+
+To load images into Model Builder, you need to provide it with the path to a single top-level directory. This top-level directory contains subfolders of a number equal to each of the categories to predict. Each subfolder contains the image files belonging to the respective category.  
+In the folder structure illustrated below, the top-level directory is *flower_photos*. There are five subdirectories corresponsing to the categories to predict: daisy, dandelion, roses, sunflowers and tulips. Each of these subdirectories contains images belonging to their respective category.
```
Comment (Youssef1313) [inline docs/machine-learning/how-to-guides/load-data-model-builder.md]:
```suggestion
In the folder structure illustrated below, the top-level directory is *flower_photos*. There are five subdirectories corresponsing to the categories to predict: *daisy*, *dandelion*, *roses*, *sunflowers*, and tulips. Each of these subdirectories contains images belonging to their respective category. 
```

File: docs/machine-learning/how-to-guides/load-data-model-builder.md [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+    |       145303599_2627e23815_n.jpg
+    |       147804446_ef9244c8ce_m.jpg
+    |       
+    \---tulips
+            100930342_92e8746431_n.jpg
+            107693873_86021ac4ea_n.jpg
+            10791227_7168491604.jpg  
+```
+
+
```
Comment (Youssef1313) [inline docs/machine-learning/how-to-guides/load-data-model-builder.md]:
```suggestion
```

**Review by Youssef1313 — CHANGES_REQUESTED** [review]

**Comment by luisquintanilla** [comment]
Work for this was merged as part of #15472

**Pull request closed without merging**